generator/osm_translator.hpp

```diff
@@ -374,6 +383,13 @@ class OsmToFeatureTranslator
     // Key point here is that IsDrawableLike and RemoveNoDrawableTypes
     // work a bit different for GEOM_AREA.
 
+    if (IsCityBoundary(params))
```

Comment:
Please move this above the comment to IsDrawableLike.

Reply:
Done